src/Skeleton/PostCreateProject.php

```diff
@@ -8,326 +8,13 @@ class PostCreateProject
 {
     public static function run(Event $event): void
     {
-        self::pinVolta();
-        self::runNpmInstall($event);
-        self::installNpmPackages($event);
-        self::installFrameworkStylePackage($event);
-        self::reconfigureWebpack($event);
         self::createAssets($event);
         self::reconfigureApplication($event);
         self::cleanupFiles($event);
         self::cleanup($event);
-        self::runNpmBuild($event);
         self::dumpInitialTranslations($event);
-    }
-
-    private static function runNpmInstall(Event $event): void
-    {
-        $io = $event->getIO();
-        $io->info('Run `npm install`');
-
-        $output = shell_exec('npm install');
-        if ($io->isVerbose()) {
-            $io->write($output);
-        }
-    }
-
-    private static function installNpmPackages(Event $event): void
-    {
-        $io = $event->getIO();
-        $io->notice('Install required NPM packages');
-
-        $packages = [
```

Comment:
Hier gaan we wel nog iets op moeten verzinnen.
Een mogelijke oplossing zou zijn om hier specifieke docker-images voor te gebruiken die we kunnen gebruiken in CI. Maar voor lokaal weet ik het nog niet zo goed.

Reply:
Fixed in https://github.com/sumocoders/application-skeleton/pull/137